internal/kldcontracts/rest2eth.go

```diff
@@ -190,6 +190,7 @@ type restCmd struct {
 	deployMsg *kldmessages.DeployContract
 	body      map[string]interface{}
 	msgParams []interface{}
+	option		string
```

Comment:
if we do go with `options` as the input, should this be a slice instead of a single string?

Reply:
rest of the internal calls seem to treat the input as variadic.. hence this looks out of place

Reply:
sorry making it variadic was mainly to not require that as a parameter, it'll only be a single parameter for all the different uses (see the new comment with the table of JSON RPC calls that uses such a "call option" parameter